src/components/EditorContainer.tsx

```diff
@@ -0,0 +1,91 @@
+import useInput from '../hooks/useInput';
+import { useRecoilState } from 'recoil';
+import { PostListState } from '../recoil/recoil';
+import { Container, Header, CenteringWrapper } from '../GlobalStyle';
+import useNavigateHome from '../hooks/useNavigateHome';
+import {
+  Form,
+  PostHandleButton,
+  Title,
+  Content,
+  InputWrapper,
+} from '../styles/EditorStyle';
+import getInputDate from '../hooks/getInputDate';
+import useRouting from '../hooks/useRouting';
+import React from 'react';
+
+function EditorContainer() {
+  const { inputText, onInputChange, reset } = useInput();
+  const [postList, setPostList] = useRecoilState(PostListState);
+
+  const navigateHome = useNavigateHome();
+  const numbering = useRouting();
+
+  const [postObj] = postList.filter((post) => post.postNum === numbering);
+
+  const savedTitle = numbering ? postObj.title : '';
+  const savedContent = numbering ? postObj.content : '';
+  const newDate = getInputDate();
+
+  const handleNewPost = (e: React.SyntheticEvent) => {
+    e.preventDefault();
+
+    if (
+      numbering
+        ? inputText.title != '' || inputText.content != ''
+        : inputText.title != '' && inputText.content != ''
```

Comment:
정규식이나 공백 제거 함수를 사용해 공백을 입력했을 때도 처리해주면 좋을 것 같아요!
```suggestion
        ? inputText.title.trim() || inputText.content.trim()
        : inputText.title.trim() && inputText.content.trim()
```

Reply:
우와,,,정규식 굿,,,